schemas/update/update_extension_schema_v1.json

```diff
@@ -63,22 +96,49 @@
     "updateStep": {
       "type": "object",
       "properties": {
-        "action": { "const": "update" },
-        "file": { "type": "string" },
-        "key": { "type": "string" },
-        "old_value": { "type": "string" },
-        "new_value": { "type": "string" }
+        "action": {
+          "const": "update",
+          "description": "The action type, which is 'update' for this step"
+        },
+        "file": {
+          "type": "string",
+          "description": "The relative path of the file to update the key-value pair in"
+        },
+        "key": {
+          "type": "string",
+          "description": "The key to update"
+        },
```

Comment:
We'll need to update this one. We'd most likely only support `setting.id` in the release. We'd hilight that in the dev docs but do you think it'd be good to add a note here as well?